src/components/passphraseCreation/index.test.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import { stub } from 'sinon';
+import PassphraseCreation from './index';
+import CreateFirst from './../passphrase/create/create';
+import * as checkDevice from '../../utils/checkDevice';
+import * as passphraseUtils from './../../utils/passphrase';
+
+describe('Passphrase Creation', () => {
+  let wrapper;
+  const events = {};
+  const props = {
+    t: key => key,
+    prevStep: () => {},
+    nextStep: () => {},
+  };
+
+  beforeEach(() => {
+    window.addEventListener = (name, event) => {
+      events[name] = event;
+    };
+    stub(passphraseUtils, 'generatePassphrase').returns('wagon stock borrow episode laundry');
```

Comment:
I would import and use https://github.com/LiskHQ/lisk-hub/blob/afff8c5684d00ad8ec74fccb8b708de954666895/test/constants/accounts.js#L3 to make it more explainatory for a newcommer what value is expected as a passphrase.